docs/API.md

```diff
@@ -203,10 +201,16 @@ bucketStream emits Object with the format:-
 **Example**
 
 ```js
-minioClient.listBuckets(function (err, buckets) {
-  if (err) return console.log(err)
-  console.log('buckets :', buckets)
-})
+const listBuckets = async () => {
```

Comment:
how about top level await in this file？

Reply:
yes. updated . thank you. 
i have added note to the specific method. we can update to be global once the complete migration is done.